apps/frontend/src/components/cards/EvaluationInfo.vue

```diff
@@ -63,6 +85,21 @@ export default class EvaluationInfo extends Vue {
     return EvaluationModule.evaluationForFile(this.file_object);
   }
 
+  convertGroupsToIVuetifyItems(groups: IGroup[]): IVuetifyItems[] {
+    return groups.map((group) => {
+      return {
+        text: group.name,
+        value: group.id
+      }
+    })
+  }
```

Comment:
This code should be de-duplicated with the code in `EditEvaluationModal`, either by putting it in a mixin or using the composition API.